pages/pipelines/artifacts.md.erb

```diff
@@ -27,60 +70,91 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
-## Uploading artifacts from scripts
+## Use the Buildkite agent to upload artifacts
+
+Within a build, run the `buildkite-agent artifact upload` command to upload artifacts from a script.
+The agent's `upload` command arguments are one or more file paths and [glob patterns](/docs/agent/v3/cli-artifact#uploading-artifacts-artifact-upload-glob-syntax).
 
-Upload artifacts from your build scripts with the `buildkite-agent artifact` command.
-The following example uploads `build.tar.gz` from the `pkg` directory to `pkg/build.tar.gz`:
+The following example uploads a `build.tar.gz` file from the `pkg` directory:
 
-```bash
+```shell
 buildkite-agent artifact upload pkg/build.tar.gz
 ```
 
-For full documentation of the `buildkite-agent artifact upload` command see the [buildkite-agent artifact upload](/docs/agent/v3/cli-artifact#uploading-artifacts) documentation.
-
+The `buildkite-agent artifact upload` command supports several options and environment variables.
+For complete usage instructions, read the [`buildkite-agent artifact upload`](/docs/agent/v3/cli-artifact#uploading-artifacts) documentation.
 
 ## Downloading artifacts
 
-You can download artifacts created by a build job, from inside a running build, using the `buildkite-agent artifact download` command. For example, to download all artifacts in the `logs` directory (including subdirectories, which will be recreated) to `local-logs/logs/`:
+When you upload an artifact, Buildkite records the file's contents, the complete path the file was uploaded from, and details of the build step it originated from, so you can retrieve artifacts by name, path, or build.
+
+You can download artifacts by running the `buildkite-agent artifact download` command during a build or by making a request to the artifacts REST API.
+
+## Use the Buildkite agent to download artifacts
```

Comment:
I used "Use" here for some symmetry with the existing page title. Do you want to see that change too?

Reply:
I'm fine for it to change, and ok to leave as is. :-p 
If we do rename the page we should add a redirect in `config/routes.rb`

Reply:
I revised the headings as they appear in the page (and nav). There's no redirect needed, since the actual URL hasn't changed. c7b021787960ed3f171bab58ee42ae3fae23ec13